core/src/view/HUD.java

```diff
@@ -16,18 +15,17 @@
 
 public class HUD {
     private final Model model;
-    private Stage stage;
-    private FitViewport stageViewport;
-    private IPlayerCharacter player;
+    private final Stage stage;
+    private final FitViewport stageViewport;
 
-    private int outlineSize = 4;
+    private final int outlineSize = 4;
 
-    private int healthBarWidth = 200;
-    private int healthBarHeight = 20;
+    private final int healthBarWidth = 200;
```

Comment:
Constants should be written like HEALTH_BAR_WIDTH instead of healthBarWidth

Reply:
Constants should also be static.